torsion_target/make_torsion_target_new.py

```diff
@@ -251,6 +267,13 @@ def test_ff_mol2(test_ff, mol2_fnm):
         return False, str(e), None
     return True, '', molecule_labels
 
+def screening_Hbond(mol2_fnm='input.mol2', scan_fnm ='scan.xyz'):
```

Comment:
This is a nice utility function and can be expanded later. A few suggestions:
1. The signature can be `def check_hbond(scan_fnm, top_fnm=None):`. This way it's more clear what we need and it's consistent with `mdtraj.load()`.

2. If this is the only place `mdtraj` is used, let move the import here so it'll not be required when later another option is provided. I think the short name `md` is too common as it often stands for "medium" or other things.
```import mdtraj```

3. Here I think the only information we need is "hydrogen bond exist or not", so I think just returning the True or False is enough. In the future if we want to write down exactly what is the hydrogen bond been detected, we can use a return data structure of a list of hydrogen bonds, such as `List[ (frame_number, (d, h, a) ) ]`.

4. Because we might want to write other functions in the same format as this function, it would be nice if you can write a doc string for it.